README.md

```diff
@@ -126,46 +132,20 @@ To facilitate the standardisation of the code, a few helpers and tools have been
 
 The projects has multiple hooks configured using [husky](https://github.com/typicode/husky#readme) which will execute the following scripts: `audit`, `lint`, `build`, `test` and format your code with [eslint](https://github.com/typescript-eslint/typescript-eslint#readme) and [prettier](https://github.com/prettier/prettier).
 
-You will be required to install [git-secrets](https://github.com/awslabs/git-secrets) (_brew approach is recommended_) and DVSA [repo-security-scanner](https://github.com/UKHomeOffice/repo-security-scanner) that runs against your git log history to find accidentally committed passwords, private keys.
+You will be required to install [git-secrets](https://github.com/awslabs/git-secrets) (_brew approach is recommended_) and DVSA [repo-security-scanner](https://github.com/UKHomeOffice/repo-security-scanner) that runs against your git log history to find accidentally committed passwords or private keys.
 
 We follow the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/) when we commit code to the repository and follow the [angular convention](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum).
 
 The type is mandatory and must be all lowercase.
-The scope of your commit remain is also mandatory, it must include your ticket number and be all lowercase. The format for the ticket number can be set in the commitlint.config.js file.
+The scope of your commit remain is also mandatory, it must include your ticket number and be all lowercase. The format for the ticket number can be set in the `commitlint.config.js` file.
 
 ```js
 // Please see /commitlint.config.js for customised format
 
 type(scope?): subject
 
 // examples
-'chore(cvsb-1234): my commit msg' // pass
-'CHORE(cvsb-1234): my commit msg' // will fail
+'chore(cb2-1234): my commit msg' // pass
+'CHORE(cb2-1234): my commit msg' // will fail
 
 ```
-
-### Code standards
-
-#### Code structure
-
-Domain Drive Design diagram with Interfaces, Application, Domain layers and Infrastructure across the layers.
-
-<p align="center">
-  <img src="./docs/DDD_architecture.jpg" alt="Domain Drive Design diagram with Interfaces, Application, Domain layers and Infrastructure across the layers" >
-</p>
-
-#### Toolings
```

Comment:
I'd keep this section. Especially the sonarqube bit.

Reply:
Kept, thanks @Ian-Hodges - I've added some instructions for running SQ in docker too